assets/home/sections/get-help.js

```diff
@@ -1,22 +1,110 @@
+/**
+ * External dependencies
+ */
+import classNames from 'classnames';
+
 /**
  * WordPress dependencies
  */
 import { __ } from '@wordpress/i18n';
+import { Icon, help, lock, external } from '@wordpress/icons';
```

Comment:
I think the lock icon from the WP lib is a little different from the design.
I checked that we have the `/assets/icons/lock.svg`, which is the same as the design.

Reply:
Changed here b7e1c32d8
Also did some improvement on sizes.